plugins/ban.plug

```diff
@@ -0,0 +1,10 @@
+[Core]
+name = Ban
+module = ban
+
+[Documentation]
+description = Ban from all rooms at once
+[Python]
+version = 3
+
+[Errbot]
```

Comment:
do we need this empty section?

if so, can we get a comment on it explaining why?

Reply:
Some of the other plugins' `plug` files were using it, I just borrowed their plug file as a template.
https://github.com/coala/corobo/blob/master/plugins/answer.plug
https://github.com/coala/corobo/blob/master/plugins/explain.plug